File: server/src/main/java/com/example/hyundaiboot/dto/PushGroupDto.java
package com.example.hyundaiboot.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.*;

@NoArgsConstructor
@AllArgsConstructor
@ToString
@Getter
@Setter
public class PushGroupDto {
	@JsonProperty("push_group_id")
	String pushGroupId;
	@JsonProperty("push_group_name")
	String pushGroupName;
	@JsonProperty("is_check")
	boolean isCheck;
}

File: app/lib/dto/push_group_dto.dart
class PushGroupDto {
  String pushGroupId;
  String pushGroupName;
  bool isCheck;

  PushGroupDto(this.pushGroupId, this.pushGroupName, this.isCheck);

  @override
  String toString() {
    return 'PushGroupDto{pushGroupId: $pushGroupId, pushGroupName: $pushGroupName, isCheck: $isCheck}';
  }
}
